rgo/component.py

```diff
@@ -35,8 +35,13 @@ def __init__(self, name, git=None, distgit=None):
         self.name = name
         self.git = git
         self.distgit = distgit
+        self.requires = set(requires)
         self.cloned = False
 
+        self.state = None
+        self.done = None
+        self.success = None
```

Comment:
You've added also `self.srpm` and `self.build_id` attributes and they should be initialized here in constructor.

Reply:
Fixed!